docs/modules/ROOT/pages/create.adoc

```diff
@@ -0,0 +1,71 @@
+== How to Create a Starter Kit
+
+=== What is a Starter Kit?
+
+https://openzeppelin.com/starter-kits/[Starter Kit] is a preconfigured Ethereum project containing the best industry tools to bootstrap a dApp development process.
+
+=== Why Should I Create a Starter Kit?
+
+Anyone can create a Starter Kit and it is a great way to introduce developers to a new tool or a project, like https://github.com/OpenZeppelin/starter-kit-gsn[GSN Starter Kit].
+
+=== How to Create a Starter Kit
+
+To create an OpenZeppelin Kit you’ll need to publish a GitHub repo containing your kit project. The GitHub repo has to contain the kit.json configuration file and have a stable branch. Repo should have meaningful README.md file walking users through installation, running and building a kit. Repo should have a stable branch that is used as a source for a kit deployment. Use a stable branch only for kit releases. https://github.com/OpenZeppelin/starter-kit-tutorial[The tutorial kit] is a good example.
+
+==== Checklist
+
+- Create a GitHub repo
+- Create a stable branch on the repo
+- Add a valid configuration file to the root of the repo
+- Add desired tools and packages to the repo
+- Add meaningful README.md file to the repo
+- Push a release version to a `stable` branch
+
+
+==== Configuration file
+
+Every Starter kit includes a configuration file – `kit.json`. This file has the following structure:
+
+[source, json]
+----
+    {
+      "manifestVersion": "0.1.0",
+      "message": "More at https://github.com/OpenZeppelin/starter-kit",
+      "files": [
+        ".gitignore",
+        "LICENSE",
+        "client",
+        "contracts",
+        "migrations",
+        "package.json",
+        "solhint.json",
+        "test",
+        "truffle-config.js"
+      ],
+      "hooks": {
+        "post-unpack": "npm install && cd client && npm install"
+      }
+    }
+----
+
+.*manifestVersion*
+A version of a manifest that ensures proper handling of a kit. Always use the version provided in the documentation.
```

Comment:
I'd also clarify that they should **not** bump this version if they want to publish a new version of the kit, or clarify that it's the version of this configuration file and not of the project.